index.html

```diff
@@ -5,45 +5,364 @@
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <title>Week 1: HTML Tables</title><!-- Latest compiled and minified CSS -->
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
+   <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Lato">
+
   <style>
     .header {
-      text-align: center;
+      text-align: left;
+      margin-top: 70px;
+     
     }
+
+    .income {
+      font-color: #F2998F;
+    }
+
+    .bar {
+          height: 20px;
+          background: #F2998F;
+        }
+      
+    .pager>li.previous>a {
+      color: #000000;
+    }
+
+    .pager>li.next>a {
+      color: #000000;
+    }
+
+   table {
+      color:  #ffffff;
+      margin-top: 50px;
+   
+    }
+
+    th {
+      font-size: 16px;
+    }
+
+    body {
+      background: #357C8D;
+      color: #ffffff;
+      font-family: "Lato";
+
+    }
+
+    h1 {
+      font-family: "Lato";
+      font-size: 30px;
+    }
+    img {margin: 50px 50px;}
+  
+    h2 {
+      font-family: "Lato";
+      font-size: 20px;
+      
+    }
+
+    .table-hover tbody tr:hover td {
+      background-color: #5a5858;
+    
+    }
+
+    a {
+      color: #ffffff;
+
+    }
+
+    .btn.outline {
+      background: none;
+      padding: 6px 6px;
+      margin: 8px 8px;
+
+    }
+
+    .btn-primary.outline {
+      border: 2px solid #ffffff;
+      color: #ffffff;
+    
+    } 
+    
+    .btn-primary:hover {
+      color: #fff;
+    background-color: #5a5858;
+    border-color: #ffffff; 
+    }
+ 
+   
+
   </style>
 </head>
 <body>
   <div class="container">
     <div class="header">
-      <h1>Week 1: HTML Tables</h1>
-      <p>We'll add our data to this table.</p>
+      <div class = "col-sm-7">
+        <h1>A Glance at D.C. Neighborhoods</h1>
+        <br />
+        <p>Washington D.C.has changed rapidly in the past decade, with more young professionals moving in certain parts of the District. Using data from <a href="http://www.neighborhoodinfodc.org" target="_blank"><u><b>Neighborhood Info DC</b></u></a>, we can examine exactly how average family income and poverty rates vary among neighborhoods within the District.</p>
+        <div class="controls section">
+          <div class="btn-group" role="group" aria-label="Sort">
+            <h2>filter:</h2> 
```

Comment:
This being inside the `.btn-group` causes issues with Bootstrap's rounded corners styles, which only apply to the first child of the group. It would be better if the `h2` was outside of the group, but there's nothing stopping you from creating a new div to wrap both the `h2` and the `btn-group` to keep them together.